auslib/web/admin/views/releases.py

```diff
@@ -355,33 +329,6 @@ class ReleaseHistoryView(HistoryView):
     def __init__(self):
         super(ReleaseHistoryView, self).__init__(dbo.releases)
 
-    def _process_revisions(self, revisions):
-        self.annotateRevisionDifferences(revisions)
```

Comment:
I believe there are no other callers of `annotateRevisionDifferences`, and no other methods of significance in the `HistoryAdminView`.

Reply:
Ah, forgot it. :)
Fixed!